### PR TITLE
Issue 199

### PR DESCRIPTION
**Pull request by rbasu101**

### CODE REVIEW

**Comment by mcannon068nw** [comment]
merge conflicts fixed, ready for PR review

**Comment by mcannon068nw** [comment]
Due to potential vulnerabilities, we should make this change even if i breaks the run example query buttons. If there is not a workaround for that, we can at the very least just display the sample queries and suggest copy/paste

File: client/package.json [hunk 1]
```diff
@@ -26,12 +26,15 @@
     "create-react-app": "^5.0.0",
     "css-loader": "^6.7.1",
     "env-cmd": "^10.1.0",
+    "graphiql": "^2.4.0",
     "graphql": "^15.5.1",
+    "graphql-playground-react": "^1.7.28",
```
Comment (jsstevenson) [inline client/package.json:31]: if we're serving it all in the frontend now, we need to pull all of the stuff out of the backend

**Review by jsstevenson — CHANGES_REQUESTED** [review]

File: client/src/pages/Playground/Playground.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+      geneCategoriesWithSources {
+        name
+        sourceNames
+      }
+    }
+  }
+}
+`
+
+const fetcher = createGraphiQLFetcher({ url: 'http://localhost:3000/api/graphql'})
```
Comment (mcannon068nw) [inline client/src/pages/Playground/Playground.tsx:132]: @acoffman @jsstevenson this works locally for me, but for deployment purposes, what should this point to? (I think there is likely an environment variable setting to use here but I do not know the best implementation)

Comment (acoffman) [reply client/src/pages/Playground/Playground.tsx:132]: I would just import and use the `API_URL` const that's already available and pulled in from the env, that should work for you!

File: client/package.json (hunk 1 above)
Comment (mcannon068nw) [reply client/package.json]: this change has been made

File: server/Gemfile [hunk 3]
```diff
@@ -26,6 +26,7 @@ gem 'graphql', '~> 1.12.15'
 gem 'graphql-batch', '~> 0.4.3'
 gem 'graphiql-rails', '~> 1.7.0'
 gem 'search_object_graphql', '0.3.1'
+gem 'graphql-playground'
```
Comment (jsstevenson) [inline server/Gemfile:29]: we should be able to delete this stuff

**Review by jsstevenson — CHANGES_REQUESTED** [review]

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**